src/test/java/org/elasticsearch/test/rest/client/RestClient.java

```diff
@@ -172,10 +172,11 @@ private HttpRequestBuilder callApiBuilder(String apiName, Map<String, String> pa
                 if (restApi.getPathParts().contains(entry.getKey())) {
                     pathParts.put(entry.getKey(), entry.getValue());
                 } else {
-                    if (!restApi.getParams().contains(entry.getKey())) {
+                    if ("filter_path".equals(entry.getKey()) || restApi.getParams().contains(entry.getKey())) {
```

Comment:
does it make sense to have some static list instead of this direct check, which has a more explaining name, why this param is here - just for readability?